__tests__/encode.js

```diff
@@ -1,8 +1,46 @@
-import {encode} from '../src';
+import {encode, decode} from '../src';
 
 describe('encode', () => {
-    it('simple', () => {
-        const data = encode({});
-        console.log(data);
+    it('simple RGBA', () => {
+        const dataArray = new Uint8Array([255, 255, 255, 255, 0, 0, 0, 255, 0, 0, 0, 255, 255, 255, 255, 255]);
+        const data = encode({
+            width: 2,
+            height: 2,
+            data: dataArray
+        });
+        expect(data).toBeInstanceOf(Uint8Array);
+        const decoded = decode(data);
```

Comment:
Do you have a better idea to check the result? Seems not ideal to me to use our own decoder